relay-protocol/src/condition.rs

```diff
@@ -163,15 +163,15 @@ pub struct GlobCondition {
     /// A list of glob patterns to check.
     ///
     /// Note that this cannot be a single value, it must be a list of values.
-    pub value: GlobPatterns,
+    pub value: TypedPatterns<CaseInsensitive>,
```

Comment:
Might it be worth it to have an alias like `CaseInsensitivePatterns` for this case?

Reply:
I think this is more clear, I'd use an alias for more complex configurations changing multiple options.

Reply:
I'll put this to the discussion in our next tech meeting, maybe it'd be better to have everything named without generics.